assets/src/components/Lightbox/Lightbox.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import PhotoSwipe from '../../../../node_modules/photoswipe/dist/photoswipe.js';
+import PhotoSwipeUI_Default from '../../../../node_modules/photoswipe/dist/photoswipe-ui-default.js';
+
+// `items` should be an array of object with this shape:
+// [{ src, w, h, title }, ...]
+// See: https://photoswipe.com/documentation/getting-started.html
+export const Lightbox = ({ index, isOpen, items, onClose }) => {
+  let photoSwipeElement = useRef(null);
+
+  const photoSwipeOptions = {
+    index: index || 0,
+    closeOnScroll: false,
+    history: false
+  };
+
+  const [currentIndex, setCurrentIndex] = useState(photoSwipeOptions.index);
+
+  if (!isOpen) {
+    onClose();
+  }
+
+  useEffect(() => {
+    const photoSwipe = new PhotoSwipe(photoSwipeElement, PhotoSwipeUI_Default, items, photoSwipeOptions);
+
+    if (!photoSwipe) {
```

Comment:
I don't think this can be the case? Since it's the result of a constructor, `photoSwipe` will always be truthy.

Reply:
True, removed.